recipes/virtual_dataframe/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "virtual_dataframe" %}
+{% set version = "v0.5rc1" %}
```

Comment:
This needs to target a pre-release build: https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build

Reply:
Fixed. Now, it's time to remove rc

Reply:
Done